contents/handbook/people/team-structure/people/objectives.mdx

```diff
@@ -1,8 +1,11 @@
-* **Proposed Objective:**
-    * Grow PostHog team by 20% without compromising team happiness or our runway
-* **Proposed Key Results:**
-    * We have offers accepted by a senior product manager, 2x SREs, senior data engineer, developer advocate, and 4x full stack engineer.
-    * We stay within 5% of forecast cash burn. 
-    * Ship 3 projects that make working at PostHog more seamless for the team - app management (Okta), migrate recruitment (Workable), self-serve equity management (LTSE Equity).
+* **Objective:**
+    * Accelerate - get new hires _and_ new customers into PostHog, faster.
+* **Key Results:**
+    * People: We have offers accepted by an SRE, 5x full stack engineers, and a product designer. 
+    * People: Hit an average time from ‘Application received’ to ‘Offer made’ of 21 days. 
+    * Ops: Reduce average time to close a support ticket by 20%
+    * Ops: Publish 3 collaborative marketing projects with vendor partners
```

Comment:
Does "Ops: Publish 3 collaborative marketing projects with vendor partners" make sense as part of this team's work?

Do we want any KRs about existing employee happiness/productivity etc.?

Reply:
The motivation for the marketing-type KR was that we wanted Ops to also spend a quarter thinking about ways they can contribute to growth (vs. last quarter where we were really focused on improving processes/systems). 

So it's a bit of an unusual one, but we thought it would be helpful for Ops to at least be thinking about how they can help PostHog accelerate growth as well. 

We tried the last couple of quarters having some variation on team NPS-type OKRs and ended up removing them as we couldn't track week to week how we were doing. My view is increasingly that team happiness shouldn't be this team's objective, as the responses we received to surveys were out of this team's control (e.g. salary, tasks they're working on, management relationships etc.)

Reply:
Makes sense thanks